sdk/communication/AzureCommunicationChat/Source/Generated/AzureCommunicationChatClient.swift

```diff
@@ -85,7 +85,8 @@ public final class AzureCommunicationChatClient: PipelineClient, PageableClient
         )
     }
 
-    public lazy var azureCommunicationChatService = AzureCommunicationChatService(client: self)
+    public lazy var chat = Chat(client: self)
+    public lazy var chatThreadOperation = ChatThreadOperation(client: self)
```

Comment:
Why not simply call this `thread`?

Reply:
Still this question.

Reply:
This is autogenerated so I'm not sure if its an autorest thing or something we can control the naming of from our swagger. chatThreadOperation maps to ChatThreadClient in the convenience layer (for operations within a particular thread)

Reply:
Oh, I thought this was in the convenience layer. In that case, no worries.